### PR TITLE
SCRIPTS: Added Fires of Discontent

### PR DESCRIPTION
**Pull request by Zenny89**

### CODE REVIEW

File: scripts/zones/Metalworks/npcs/Iron_Eater.lua [hunk 1]
```diff
@@ -71,26 +77,29 @@ end;
...
+            player:messageSpecial(ITEM_CANNOT_BE_OBTAINED,182);
+            player:setVar("Flagbastok",1);
+        else    
+            player:addItem(182);
+            player:messageSpecial(ITEM_OBTAINED,182);
+        end
+        player:setVar("BASTOK92",0);
+        player:completeMission(BASTOK,WHERE_TWO_PATHS_CONVERGE);
+        player:setRank(10);
+        player:addGil(100000);
```
Comment (xdemolish) [inline scripts/zones/Metalworks/npcs/Iron_Eater.lua]: since you're GIL_RATE'ing anyways...

**Comment by Zenny89** [comment]
GIL_RATE'd

**Comment by xdemolish** [comment]
don't forget to require("scripts/globals/settings"); when GIL_RATE and other crap defined in there's used

**Comment by Zenny89** [comment]
Done - I still got the correct gil rate without requiring the settings though (when my rate was 1). Don't know if it'd still work when gil rate is different.

**Pull request merged**